cliconfig/credentials/default_store_unix.go

```diff
@@ -2,4 +2,4 @@
 
 package credentials
 
-const defaultCredentialsStore = ""
+const defaultCredentialsStore = "secretservice"
```

Comment:
Any reason for calling it "secretservice", and not, for example "gnome-libsecret" ?